docs/en/setup/plugins/queue_mmap-queue.md

```diff
@@ -15,6 +15,9 @@ flush_period: 1000
 flush_ceiling_num: 10000
 # The max size of the input event. Default value is 20k.
 max_event_size: 20480
+
+# The partition count of queue.
+partition_count: 1
```

Comment:
mmap queue supports partitions?

Reply:
Yes, It supports. I wrap a partitioned queue to manage all sub-queue(memory or mmap).